README.md

```diff
@@ -65,16 +65,17 @@ If you are using the STARR-OMOP dataset from Stanford (which uses the OMOP CDM),
 1. Download your STARR-OMOP dataset to `[PATH_TO_SOURCE_OMOP]`.
 2. Convert STARR-OMOP => MEDS using the following:
 ```bash
-femr_stanford_omop_fixer [PATH_TO_SOURCE_OMOP] [PATH_TO_SOURCE_OMOP]
-
 # Convert OMOP => MEDS data format
 meds_etl_omop [PATH_TO_SOURCE_OMOP] [PATH_TO_OUTPUT_MEDS]
```

Comment:
Let's do [PATH_TO_OUTPUT_MEDS]_raw

And then [PATH_TO_OUTPUT_MEDS] for the output of the fixer. Less chance of issues that way.

Reply:
Fixed! @EthanSteinberg